### PR TITLE
Add guidance for setting up forks

### PR DESCRIPTION
**Pull request by karanbirsingh**
This PR adds initial documentation about git workflow - both [web](https://github.com/Microsoft/accessibility-insights-web/pull/381) and windows are modeled after OF. May be able to consolidate these separate sources but for now this works.

See preview here: https://github.com/Microsoft/accessibility-insights-windows/pull/192/files?short_path=73ea53d#diff-73ea53d2bfcb766b349c692c9bc78185

### CODE REVIEW

File: docs/GitBranchSetup.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+<!--
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+-->
+
+## Git branch setup
+
+This document describes how to set up your development environment and contribute changes to
+[accessibility-insights-windows](https://github.com/Microsoft/accessibility-insights-windows). This document assumes basic working knowledge
+with Git and related tools. We are providing instructions specific to this project.
```
Comment (RobGallo) [inline docs/GitBranchSetup.md]:
```suggestion
of Git and related tools. We are providing instructions specific to this project.
```

File: docs/GitBranchSetup.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+## Creating your own fork
+
+If you wish to contribute changes back to the [accessibility-insights-windows](https://github.com/Microsoft/accessibility-insights-windows)
+repository, start by creating your own [Fork](https://help.github.com/en/articles/fork-a-repo) of the repository. This will keep the
```
Comment (RobGallo) [inline docs/GitBranchSetup.md]: How about, "keep down the number of branches on this repository"?

File: docs/GitBranchSetup.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+count. In your own fork, you can create as many branches as you like.
+
+-   Navigate to [GitHub](https://github.com/) with a browser and log in to your GitHub account. For the sake of this document, let's assume your username is **adacat**.
+-   Navigate to the [accessibility-insights-windows](https://github.com/Microsoft/accessibility-insights-windows) repository in the same browser session.
+-   Click on the **Fork** button at the top right corner of the page.
+-   Create the fork under your account. Your GitHub profile should now show **accessibility-insights-windows** as one of your repositories.
+-   Create a folder on your device and clone your fork of the **accessibility-insights-windows** repository. e.g. `https://github.com/adacat/accessibility-insights-windows`. Notice how your GitHub username is in the repository location.
+
+```
+> git clone https://github.com/adacat/accessibility-insights-windows
```
Comment (RobGallo) [inline docs/GitBranchSetup.md]: Should we actually have an AdaCat user/repo?

File: docs/GitBranchSetup.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+origin  https://github.com/adacat/accessibility-insights-windows (push)
+```
+
+-   Map the primary **accessibility-insights-windows** as the upstream remote
+
+```
+> git remote add upstream https://github.com/Microsoft/accessibility-insights-windows
+```
+
+-   Now running `git remote -v` should show the upstream repository also
```
Comment (RobGallo) [inline docs/GitBranchSetup.md]:
```suggestion
-   Now, running `git remote -v` should show the upstream repository also
```

File: docs/GitBranchSetup.md (hunk 3 above)
Comment (karanbirsingh) [reply docs/GitBranchSetup.md]: @RobGallo we do have one but I didn't realize the docs spell it wrong. It should be ada-cat. You can see it [here](https://github.com/ada-cat). I will update.

**Comment by karanbirsingh** [comment]
@RobGallo updated

**Comment by karanbirsingh** [comment]
> I've made some suggestions and I'm okay approving the PR, but why are we duplicating existing GitHub documentation like this.

There's some overlap, but it's nice to be able to copy-paste repo-specific commands. I prefer the scenario-based instructions here for keeping a fork in sync. If it doesn't hamper the expert user and provides a bit more detail for someone new I think it's worthwhile.

**Pull request merged**